r_examples/r_batch_transform/r_xgboost_batch_transform.ipynb

```diff
@@ -6,7 +6,13 @@
    "source": [
     "<h1>Batch Transform Using R with Amazon SageMaker</h1>\n",
     "\n",
-    "**Note:** You will need to use R kernel in SageMaker for this notebook.\n",
+    "**Read before running this notebook:**\n",
+    "\n",
+    "- This sample notebook has been updated for SageMaker SDK v2.0. \n",
+    "- If you are using SageMaker Notebook instances, select `R` kernel for the notebook. If you are using SageMaker Studio notebooks, you will need to create a custom R kernel for your studio domain. Follow the instructions in this blog post to create and attach a custom R kernel. \n",
+    "    - [Bringing your own R environment to Amazon SageMaker Studio](https://aws.amazon.com/blogs/machine-learning/bringing-your-own-r-environment-to-amazon-sagemaker-studio/)\n",
+    "    \n",
+    "**Summary:**\n",
     "\n",
     "This sample Notebook describes how to do batch transform to make predictions for an abalone's age, which is measured by the number of rings in the shell. The notebook will use the public [abalone dataset](https://archive.ics.uci.edu/ml/datasets/abalone) hosted by [UCI Machine Learning Repository](https://archive.ics.uci.edu/ml/index.php).\n",
```

Comment:
We have it hosted here:
s3://sagemaker-sample-files/datasets/tabular/uci_abalone/
Can you get it from there instead of the public location?

Reply:
Done. Changed the code to pull from S3 data instead of UCI repo.
```suggestion
    "This sample Notebook describes how to do batch transform to make predictions for an abalone's age, which is measured by the number of rings in the shell. The notebook will use the public [abalone dataset](https://archive.ics.uci.edu/ml/datasets/abalone) originally from [UCI Machine Learning Repository](https://archive.ics.uci.edu/ml/index.php).\n",
```

The cell immediately after this will change to:
```
library(readr)
data_file <- 's3://sagemaker-sample-files/datasets/tabular/uci_abalone/abalone.csv'
abalone <- read_csv(file = sagemaker$s3$S3Downloader$read_file(data_file, sagemaker_session=session), 
                    col_names = FALSE)
names(abalone) <- c('sex', 'length', 'diameter', 'height', 'whole_weight', 'shucked_weight', 'viscera_weight', 'shell_weight', 'rings')
head(abalone)
```